src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+$primary-color: #060b35;
+$secondary-color: #616070;
+$button-color: #00acdc;
+$number-of-stars: 4;
+$card-elements-width: 166px;
+
 body {
   margin: 0;
+
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 18px;
+  color: $primary-color;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+
+  &__photo {
+    background-image: url(../images/imac.jpeg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    width: $card-elements-width;
```

Comment:
there is no need to specify the width pixel by pixel, if necessary, it is better to spread it out - '100%'
and so on everywhere